src/WiFiScanManager.cpp

```diff
@@ -84,16 +84,16 @@ void _iterateChannel() {
 }
 
 void _evScanCompleted(arduino_event_id_t event, arduino_event_info_t info) {
-  std::uint16_t numNetworks = WiFi.scanComplete();
+  std::int16_t numNetworks = WiFi.scanComplete();
   if (numNetworks < 0) {
     _handleScanError(numNetworks);
     return;
   }
 
-  for (std::uint16_t i = 0; i < numNetworks; i++) {
+  for (std::int16_t i = 0; i < numNetworks; i++) {
```

Comment:
Genuine question, why is "an amount" a signed number? Surely it can't be negative?